gocron.go

```diff
@@ -454,13 +455,14 @@ func (s *Scheduler) Err() error {
 
 // RunPending runs all the jobs that are scheduled to run.
 func (s *Scheduler) RunPending() error {
+	s.mu.Lock()
```

Comment:
Just because the line below is a boolean, does not make access to it threadsafe.